lib/routes/publico/ciencias.ts

```diff
@@ -0,0 +1,50 @@
+import { Route } from '@/types';
+import got from '@/utils/got';
+import { load } from 'cheerio';
+
+import getItems from './items-processor';
+
+export const route: Route = {
+    path: '/ciencias/:subsection?',
+    categories: ['traditional-media'],
+    example: '/ciencias',
```

Comment:
```suggestion
    example: '/publico/ciencias',
```
example should have namespace prepended